static_src/constants.js

```diff
@@ -22,6 +22,8 @@ const entityHealth = keymirror({
   warning: null,
   // Something is definitely not right and requires some attention
   error: null,
+  // Entity process was successful
+  finish: null,
```

Comment:
The styles for `finish` are already in the style library. Im happy to use success, but then we need to make a number of additional changes.

Do you think we should do that?